sites/game-client/map-renderer.mjs

```diff
@@ -58,6 +58,7 @@ class MapRenderer {
     // 'playerImage' and 'playerName' are layers defined and used only by MapRenderer
     this.registerCustomizedLayerToDraw(LAYER_PLAYER_IMAGE.zIndex, LAYER_PLAYER_IMAGE.layerName, '_drawManyCharacterImage', this.gameState.players);
     this.registerCustomizedLayerToDraw(LAYER_PLAYER_NAME.zIndex, LAYER_PLAYER_NAME.layerName, '_drawManyCharacterName', this.gameState.players);
+    this.registerCustomizedLayerToDraw(1, 'object');
```

Comment:
Not sure if this is custom or we should make it just like 'ground'. For this PR I think we can just leave it as is.

Reply:
Got it,  maybe we need to define why kind of layers we need